lib/laziness.rb

```diff
@@ -4,7 +4,7 @@
 require 'laziness/auth'
 require 'laziness/channel'
 require 'laziness/chat'
-require 'laziness/client'
+require 'laziness/web_client'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.